src/services/navigationBar.ts

```diff
@@ -176,23 +190,35 @@ namespace ts.NavigationBar {
             }
         }
 
+        function hasNamedFunctionDeclarations(nodes: NodeArray<Statement>) {
+            if (forEach(nodes, s => s.kind === SyntaxKind.FunctionDeclaration && !isEmpty((<FunctionDeclaration>s).name.text))) {

```

Comment:
I created for-of-loop here, because `forEach` is returning a truthy value or `undefined`. With a for-of I can return `boolean`.